### PR TITLE
Added Button

### LINKED ISSUE #2: Button
- src/components/pre-live/Button.jsx
- props: text, onClick
- the onClick will be on the last word in the text
image

### PR DESCRIPTION
**Pull request by seanquiambao**
image

### CODE REVIEW

File: src/components/static/prelive/Button.jsx [hunk 1]
```diff
@@ -0,0 +1,23 @@
+"use client";
```
Comment (shahdivyank) [inline src/components/static/prelive/Button.jsx]: u dont need use client

File: src/components/static/prelive/Button.jsx [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+import { LuArrowUpRight } from "react-icons/lu";
+
+const Button = ({ text, onClick }) => {
+  const words = text.split(" ");
```
Comment (shahdivyank) [inline src/components/static/prelive/Button.jsx]: why do u need to split and then join later?

File: src/components/static/prelive/Button.jsx [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+  return (
+    <div className="text-2xl font-bold items-center">
+      <p>
+        {words.join(" ")}{" "}
+        <span
+          className="text-cutie-orange cursor-pointer underline"
+          onClick={onClick}
+        >
+          {last}
+        </span>{" "}
```
Comment (shahdivyank) [inline src/components/static/prelive/Button.jsx]: remove trailing space

File: src/components/static/prelive/Button.jsx [hunk 4]
```diff
@@ -0,0 +1,23 @@
+"use client";
+import { LuArrowUpRight } from "react-icons/lu";
+
+const Button = ({ text, onClick }) => {
```
Comment (shahdivyank) [inline src/components/static/prelive/Button.jsx]: only the Mentor should be passed in, not anything else

**Review by menthy-wu — APPROVED** [review]

**Pull request merged**